plugins/scaffolder-common/src/Template.v1beta3.schema.json

```diff
@@ -139,6 +139,24 @@
                 }
               ]
             },
+            "presentation": {
```

Comment:
Hey @acierto 👋🏻 

Is this also missing the `buttonLabels` field? (See [here](https://github.com/backstage/backstage/blame/master/plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx#L184-L190))

Was trying to get this functionality to work on my backstage instance and I can't, was thinking this might be the reason why. 😄

Reply:
Thanks! I was using the `createButtonText` when I needed the `reviewButtonText` 🤦🏻